dubbo-serialization-extensions/dubbo-serialization-fury/src/main/java/org/apache/dubbo/common/serialize/fury/dubbo/FuryObjectInput.java

```diff
@@ -40,7 +40,7 @@ public FuryObjectInput(Fury fury, MemoryBuffer buffer, InputStream input) {
 
   @Override
   public Object readObject() {
-    return fury.deserialize(new FuryInputStream(input, buffer.size()));
+    return fury.deserializeJavaObjectAndClass(new FuryInputStream(input, buffer.size()));
```

Comment:
Could we use `BlockedStreamUtils.deserialize(fury, output, obj);`  and `BlockedStreamUtils.serialize(fury, output, obj);` instead?  

`FuryInputStream` will buffer data internally for speed, which may read more data than needed